core/org.osate.aadl2.modelsupport/src/org/osate/aadl2/modelsupport/EObjectURIWrapper.java

```diff
@@ -63,27 +83,139 @@
 public class EObjectURIWrapper {
```

Comment:
The old constructor was public, so it should be deprecated before removing it.

Reply:
Put back the old constructor.